### ISSUE TITLE
Mapnik and python3 - help needed !

### ISSUE THREAD

**Issue by olivierdalang** [mapnik/mapnik-support#66]
Hi !

Sorry for crossposting (to the google group which seems a bit dead)...

Is it possible to use mapnik under python 3 ? I read it's supported, but didn't find a way to install, nor much discussion about it.

Mapnik is very easy to install under python2.7 with pip. But there seem to be no package in PyPI for python 3.x.

I tried installing it with pip and then copy the installation over to python3.4, but of course it doesn't work (errors on import mapnik).

Then, I found that there a python3-mapnik package in Ubuntu 15.04+'s repositories. I tried to install it under 15.04, but get errors :

> > > import mapnik
> > > Traceback (most recent call last):
> > >   File "<stdin>", line 1, in <module>
> > >   File "/usr/lib/python3/dist-packages/mapnik/**init**.py", line 74, in <module>
> > >     from ._mapnik import *
> > > ImportError: /usr/lib/x86_64-linux-gnu/libboost_python-py27.so.1.58.0: undefined symbol: PyClass_Type

In short : is there a not-too-hard way to install it under python3.4 ? If not, is a Python3 package planned ? (the website says "We plan on providing prebuilt binaries soon for Mapnik 3.0.9" but doesn't give a date nor if python 3 will be supported).

Thanks a lot !

Olivier

**Comment by olivierdalang** [comment]
Anyone on this ? Is this the best place to get support ?

**Comment by artemp** [comment]
@olivierdalang - 

```
/usr/lib/x86_64-linux-gnu/libboost_python-py27.so.1.58.0
```

the name implies it's a boost 1.58 lib built against python 2.7 ! You need to have (build from source?) `libboost_python-py3x.so.1.58.0`

**Comment by totalspam** [comment]
I have the same issue. Is anyone could help?
